guide/12-enrich-data-with-thematic-information/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -11,139 +11,576 @@
     "# Part 1 - Introduction to GeoEnrichment"
```

Comment:
<h3><strong>Enriching Study Areas </strong></h3><p> </p><p>The GeoEnrichment <strong>enrich</strong> capability adds location intelligence to your data by providing facts about a location or an area. Using GeoEnrichment, you can get information about the people and places in a specific area, or within a certain distance or drive time from a location. It enables you to query and use information from a large collection of datasets, including data regarding population, income, housing, consumer behavior, and the natural environment.</p><p> </p><p>This enables you to answer questions about locations that you can't answer with maps alone. For example, what kind of people live here? What do people like to do in this area? What are their habits and lifestyles?</p><p> </p><p>GeoEnrichment makes your analysis more powerful by adding demographic variables in a geographic context. Furthermore, these variables can be accessed at multiple standard geographic resolutions based on the jurisdictional area.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>